ghost/collections/src/CollectionsService.ts

```diff
@@ -174,47 +174,83 @@ export class CollectionsService {
     subscribeToEvents() {
         this.DomainEvents.subscribe(PostDeletedEvent, async (event: PostDeletedEvent) => {
             logging.info(`PostDeletedEvent received, removing post ${event.id} from all collections`);
-            await this.removePostFromAllCollections(event.id);
+            try {
+                await this.removePostFromAllCollections(event.id);
+            } catch (err) {
+                logging.error({err});
```

Comment:
I think in most instances we don't use this syntax 🤔  Just the `logging.error(err)` or wrapping it in a GhostError is the way to log